packages/mui-joy/src/Button/Button.test.js

```diff
@@ -0,0 +1,64 @@
+import * as React from 'react';
+import { expect } from 'chai';
+import { describeConformance, createRenderer } from 'test/utils';
+import Button, { buttonClasses as classes } from '@mui/joy/Button';
+import { ThemeProvider } from '@mui/joy/styles';
+
+describe('Joy <Button />', () => {
+  const { render } = createRenderer();
+
+  describeConformance(<Button>Conformance?</Button>, () => ({
+    render,
+    classes,
+    ThemeProvider,
+    refInstanceof: window.HTMLButtonElement,
+    muiName: 'JoyButton',
```

Comment:
Oops, we got the name wrong no?

```suggestion
    muiName: 'MuiButton',
```

Reply:
At first, I would like to separate Joy from MD because the result from `useThemeProps` might conflict with the upper theme in the docs.

However, I think it might be a better way to use the same thing as MD and see what issues we encounter.

will use `MuiButton` instead.